portal/messages/en.json

```diff
@@ -268,8 +276,10 @@
       "time-remaining": "Time remaining",
       "tx": "Tx",
       "unlock": "Unlock",
+      "unlocked": "Unlocked",
       "unlocks-on": "Unlocks on {date}"
-    }
+    },
+    "unstake-successful": "Unlock successful"
```

Comment:
This is related to https://github.com/hemilabs/ui-monorepo/pull/1490/files#r2337939123

Is it unstake (in the object key) or unlock (in the object value)?

The contract uses "Withdraw". So there are 3 names for basically the same operation

We cannot change "withdraw". Ideally, we would match the contract, but on some occasions it's ok if they are named differently. I'd stick with either "Unstake" or "unlock" everywhere. If the UI uses "Unlock", maybe we should use that one instead of Unstake.

What are your thoughts on this?